README.md

```diff
@@ -480,6 +482,8 @@ ull}}]],
 
 ## `RegisterClient`
 
+`RegisterClient` is an object that responds to several methods. 
+
```

Comment:
This doesn't give me useful information about _what_ is `RegisterClient` for and _why_ should I care. I would try to go on the lines of:

> [`RegisterClient`](lib/register_client.rb) is the main entrypoint of this library. It allows you to manage a remote Register using any backend that implements [`DataStore`](lib/data_store.rb). This library provides an in-memory implementation of `DataStore` to let you get started quickly but you should consider implementing a the interface for your backend of choice (e.g. PostgreSQL).

@karlbaker02 please double-check that I'm not saying anything wrong here.

Reply:
A few small things:

Technically, `RegisterClientManager` is the main entrypoint of this library, from which you get back a `RegisterClient` instance after having called `get_register` on the manager. 

It is also the `RegisterClientManager` which creates the `InMemoryDataStore` and passes it to the `RegisterClient` - passing your own implementation of `DataStore` to the manager will override this behaviour and use your data store instead.

Looks good otherwise!

Reply:
That makes sense to me but shouldn't we restructure the guidance a little to better reflect the fact that `RegisterClientManager` is technically the main entrypoint of the library?

Reply:
I think we should, yes.

Reply:
In which case, I'd like to delete this sentence and open a new PR for something better reflective:

> `RegisterClient` is an object that responds to several methods.